src/flamegpu/sim/Simulation.cu

```diff
@@ -288,9 +293,9 @@ int Simulation::checkArgs(int argc, const char** argv) {
         if (checkArgs_derived(argc, argv, i)) {
             continue;
         }
-        fprintf(stderr, "Unexpected argument: %s\n", arg.c_str());
-        printHelp(argv[0]);
-        return false;
+        // Warning if not in QUIET verbosity or if silnce-unknown-args is set
+        if (!(config.verbosity == flamegpu::Verbosity::Quiet || config.silence_unknown_args))
```

Comment:
For this to work, a user has to specify `silence_unknown_args`, programmatically or on command line, prior to the unknown argument. Otherwise, we'd need to double parse commandline.

Bit grim.

Reply:
Maybe worth adding a note about that quirk to the docs?

Reply:
Its in the docs and the help. "Silence warnings for unknown arguments passed **after** this flag."